docs/transparent-validation/rte/v1.1.0/flashing-guide.md

```diff
@@ -0,0 +1,97 @@
+# Flashing guide
+
+This document describes how to set up external flashing on some devices using
+RTE.
+
+## NS5x/7x 12th Generation
+
+### Prerequisites
+
+* [Prepared RTE](../v1.1.0/quick-start-guide.md)
+* WSON8 probe - can be bought from China on Aliexpress or eBay.
```

Comment:
Does the information about, where the probe might be bought necessary?

Reply:
No I guess, I removed it, I copied it from another document: https://docs.dasharo.com/variants/msi_z690/recovery/